README.md

```diff
@@ -26,6 +37,8 @@ Today `ubuntu-server-netboot` needs to run on Ubuntu or another Debian derivativ
  - pxelinux (x86-only)
  - syslinux-common (x86-only)
 
+This script is tested with Bionic and above.
```

Comment:
*nit* - can you say Ubuntu 18.04 ('bionic') and above? I'm not sure if we can expect the average community member to know Ubuntu codenames.